### PR TITLE
Sdk 1697 update native sdk versions

### PR DESCRIPTION
**Pull request by echo-branch**
## Reference
SDK-1697

## Summary
Update dependencies.
iOS to 1.45.2
Android to 5.2.7

## Motivation
Update native SDK versions. Note this increases the min iOS version to iOS 11.

## Type Of Change
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Testing Instructions
Should see the correct native SDK versions in logs.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-native-branch",
-  "version": "5.6.2",
+  "version": "5.7.0",
```
Comment (gdeluna-branch) [inline package.json:3]: For upping ios requirements, do we usually bump the major? or is 9->11 not disruptive enough to warrant?

**Comment by gdeluna-branch** [comment]
Can you update https://github.com/BranchMetrics/react-native-branch-deep-linking-attribution/blob/master/ChangeLog.md

**Review by gdeluna-branch — APPROVED** [review]

**Review by nsingh-branch — APPROVED** [review]

**Comment by RayHughes** [comment]
Has there been an update to this? Looking to get the new SDKs in production.

**Comment by echo-branch** [comment]
> Can you update https://github.com/BranchMetrics/react-native-branch-deep-linking-attribution/blob/master/ChangeLog.md
Was thinking of doing this as part of the release ticket.

**Pull request merged**